docs/CONTRIBUTING.md

```diff
@@ -1,25 +1,47 @@
 # Contributing to Endless Sky
 
-## Posting issues
+## Issues
+
+### Posting issues
 
 The [issues page](https://github.com/endless-sky/endless-sky/issues) on GitHub is for tracking bugs and feature requests. When posting a new issue, please:
 
-* Check to make sure it's not a duplicate of an existing issue.
+* Check to make sure it is not a duplicate of an existing issue.
 * Create a separate "issue" for each bug you are reporting and each feature you are requesting.
+* Choose the correct issue template and fill it out to the best of your ability.
 * Do not use the issues page for things other than bug reports and feature requests. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead.
```

Comment:
```suggestion
* Do not use the issues page for things other than what match the provided issue templates. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead. Examples of what the discussion pages would be used for include topics that are not looking for a resolution (as the point of an issue is to have it be resolved) or topics that are extremely broad and not befitting of an issue because there is no single resolution to the topic.
```

Reply:
We should probably make a balance template.